src/index.html

```diff
@@ -11,7 +11,177 @@
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset id="first" class="fieldset">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input
+            id="surname"
+            type="text"
+            autocomplete="off"
+            name="surname"
+          >
+        </div>
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input
+            id="name"
+            type="text"
+            autocomplete="off"
+            name="name"
+            maxlength="15"
+          >
+        </div>
+        <div class="form-field">
+          <label for="age">How old are You?</label>
+          <input
+            id="age"
+            type="number"
+            min="1"
+            max="100"
+            value="12"
+            name="age"
+          >
+        </div>
+        <div class="form-field">
+          <label for="date-of-birth">Full date of birth:</label>
+          <input
+            id="date-of-birth" type="date"
+            placeholder="mm/dd/yyyy" name="date-of-birth"
+          >
+        </div>
+        <div class="form-field">
+          <label for="terms-and-conditions">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="terms"
+            id="terms-and-conditions"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="mail">E-mail:</label>
+          <input
+            type="email"
+            name="mail"
+            id="mail"
+            placeholder="email@example.com"
+          >
+        </div>
+        <div class="form-field">
+          <label for="pass">Password:</label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="6"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <input
+            type="radio"
+            id="yes"
+            name="cat"
+          >
+          <label for="yes">Yes</label>
+          <input
+            type="radio"
+            id="no"
+            name="cat"
+          >
+          <label for="no">No</label>
+        </div>
+        <div class="form-field">
+          <label for="color">
+            What is your favorite color?
+            <input
+              type="color"
+              name="color"
+              id="color"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="time-for-bed">
+            What time do you go to bed?
+            <input
+              type="time"
+              name="time-for-bed"
```

Comment:
[checklist p4](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
```suggestion
              name="timeForBed
```